reports/hemang.md

```diff
@@ -34,9 +34,9 @@ email: hemangsk@gmail.com
 blog: http://hemangsk.github.io/stories.html
 activity:
  - 0:
-   - repo: coala-quickstart 
+   - repo: coala-quickstart
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.